python/mqt/debugger/dap/messages/restart_dap_message.py

```
@@ -64,12 +65,20 @@ def handle(self, server: DAPServer) -> dict[str, Any]:
         """
         server.simulation_state.reset_simulation()
         program_path = Path(self.program)
+        server.source_file = {"name": program_path.name, "path": self.program}
+        parsed_successfully = True
         code = program_path.read_text(encoding=locale.getpreferredencoding(False))
```

Ruff (0.14.10)

Boolean positional value in function call

(FBT003)